src/nxdoc/nuxeo-server/platform-services/comments.md

```diff
@@ -209,7 +209,8 @@ Annotations:
 | ---------------------------------------------- | ----------- | ----------------------------------------------------------- | ----------------------- | --------------------------------- |
 | Create an annotation on document               | POST        | `/@annotation`                                              | Annotation              | Created annotation                |
 | Fetch annotations of document                  | GET         | `/@annotation?xpath=file:content`                           | /                       | Annotation of specified blob      |
-| Fetch replies of annotations                   | GET         | `/@annotation/comments?annotationIds=...&annotationIds=...` | /                       | List of replies                   |
+| Fetch replies of annotations                   | GET         | `/@annotation/comments?annotationIds=...&annotationIds=...` | /                       | List of replies (deprecated) |
```

Comment:
Worth adding since when the API is deprecated?

Reply:
I put since 11.3, we may want to change it change is not in this promotion.